### ISSUE TITLE
g++, vim 설치 관련

### ISSUE THREAD

**Issue by mojunsang26** [kosslab-kr/Tizen-NN-Runtime#7]
저는 다 설치하고 생각나는대로 shell script를 만들어서 제대로 작동하는지 확인을 못했습니다.
잘 실행되지 않는다면 수정 바랍니다.

**Comment by timedilation** [comment]
혹시 설치하신 rpm파일들 알려주실 수 있으실까요?

**Comment by eksson** [comment]
모준상님께서 Tizen-NN-Runtime / basic-tools / 폴더에 올려 놓으신 것 같은데요.

**Comment by timedilation** [comment]
아 그렇군요 감사합니다.

**Comment by mojunsang26** [comment]
네 맞습니다. 우선 clone하신 후 rpm패키지 설치 시 dependency error발생할 경우 해당 패키지 디렉토리의 .sh파일 수정 바랍니다.
저는 현재 올라와 있는 basic-tools의 rpm외엔 사용하지 않았으니 따로 받으실 필요 없을 것 같습니다.

**Comment by timedilation** [comment]
네 감사합니다.
